string name
geometry_msgs/Pose[] waypoints
geometry_msgs/Pose[] headings